src/frontend/components/material-admin/less/inc/vendor-overrides/fullcalendar.less

```diff
@@ -184,15 +184,14 @@
     border: 0;
 
     .fc-title {
-		padding: 2px 8px;
+		padding: 2px 6px;
 		display: block;
     }
 
     .fc-time {
 		float: left;
 		background: rgba(0, 0, 0, 0.2);
 		padding: 2px 6px;
-		margin: 0 0 0 -1px;

```

Comment:
You should not change here. This is a copy version of material-admin library. It can be overridden with other versions of we want to upgrade.

Instead, you can have a custom version in the calendar for this by overriding it with the correct scope.

Reply:
No Tuan. What is in `src/frontend/components` are unmaintained dependencies, so it's our job to maintain them.

you CAN change things in this subfolder.

Reply:
Yes I've thought of that, and in the end I decided to do it this way because they are unmaintained (dead) depdencies as Michael says, so there will never be an upgrade.

Reply:
Ok. Good to know!